src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
+        if (!currentSchemaData) return undefined;
+        const {CreateStep, PathType, PathSubType, PathId, PathVersion} = currentSchemaData;
+
+        const overview: InfoViewerItem[] = [];
+
+        // for any schema type
+        overview.push({label: 'Type', value: PathType?.replace(/^EPathType/, '')});
 
-        let component =
-            currentSchemaData?.PathType && pathTypeToComponent[currentSchemaData.PathType]?.();
+        // show SubType if it's not EPathSubTypeEmpty
+        if (PathSubType !== EPathSubType.EPathSubTypeEmpty) {
+            overview.push({label: 'SubType', value: PathSubType?.replace(/^EPathSubType/, '')});
+        }
+
+        // show PathId
+        // if you get 18446744073709551615 = ffffffff = 2n ** 64n - 1n =-1 в ui64
+        // you need to ask @xeno0904 for help
+        overview.push({label: 'Id', value: PathId});
 
-        if (!component) {
-            component = <InfoViewer info={[{label: createdAtLabel, value: createdAt}]} />;
+        // show PathVersion
+        overview.push({label: 'Version', value: PathVersion});
+
+        // show created time if it's not 0
+        if (isNumeric(CreateStep) && Number(CreateStep)) {
+            overview.push({
+                label: 'Created',
+                value: dateTimeParse(Number(CreateStep))?.format('YYYY-MM-DD HH:mm'),
+            });
+        }
+
+        const {PathDescription} = currentObjectData;
+        const title = getEntityName(PathDescription);
+
+        // Table: show Partitions count = len(TablePartitions)
+        if (PathType === EPathType.EPathTypeTable) {
+            overview.push({
+                label: 'Partitions count',
+                value: PathDescription?.TablePartitions?.length,
+            });
+        }
+
+        // ColumnTable: show Partitions count = len(ColumnTableDescription.Sharding.ColumnShards)
+        if (PathType === EPathType.EPathTypeColumnTable) {
+            overview.push({
+                label: 'Partitions count',
+                value: PathDescription?.ColumnTableDescription?.Sharding?.ColumnShards?.length,
+            });
+        }
+
+        // ColumnStore: show Partitions count = len(ColumnStoreDescription.ColumnShards)
+        if (PathType === EPathType.EPathTypeColumnStore) {
+            overview.push({
+                label: 'Partitions count',
+                value: PathDescription?.ColumnStoreDescription?.ColumnShards?.length,
+            });
+        }
+
+        // ExtSubDomain: (database)
+        // show Paths (DomainDescription.PathsInside)
+        // show Shards (DomainDescription.ShardsInside)
+        if (PathType === EPathType.EPathTypeExtSubDomain) {
+            overview.push({
+                label: 'Paths count',
+                value: PathDescription?.DomainDescription?.PathsInside?.length,
+            });
+            overview.push({
+                label: 'Shards count',
+                value: PathDescription?.DomainDescription?.ShardsInside?.length,
```

Comment:
in the types ShardsInside is a string, it seems that counting it's length is incorrect